src/pyclaw/gauges.py

```diff
@@ -98,6 +98,10 @@ def read(self, gauge_id, path=None, use_pandas=False):
             # First line
             data = gauge_file.readline().split()
             self.id = int(data[2])
+            if len(data) == 8:
```

Comment:
Instead of checking the number of values can we add a parameter that specifies the number of dimensions?